opentelemetry-etw-metrics/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "opentelemetry-etw-metrics"
+version = "0.1.0"
+edition = "2021"
+description = "OpenTelemetry metrics exporter to ETW (Event Tracing for Windows)"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-etw-metrics"
+repository = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-etw-metrics"
+readme = "README.md"
+keywords = ["opentelemetry", "metrics", "etw"]
+license = "Apache-2.0"
+rust-version = "1.65"
+
+[dependencies]
+opentelemetry = { workspace = true, features = ["metrics"] }
+opentelemetry_sdk = { workspace = true, features = ["metrics", "rt-tokio"] }
+opentelemetry-proto = { workspace = true, features = ["gen-tonic", "metrics"] }
+eventheader = { version = "= 0.3.4" }
```

Comment:
Do we need this crate?

Reply:
Thanks, removed `eventheader` and `lazy_static`